Sources/Testing/Running/Runner.RuntimeState.swift

```diff
@@ -222,3 +222,12 @@ extension Test.Case {
     return try await Runner.RuntimeState.$current.withValue(runtimeState, operation: body)
   }
 }
+
+// MARK: - Event.Context convenience initializer
+
+extension Event.Context {
+  init() {
```

Comment:
Documentation?

Reply:
See #677—this `init()` won't stick around.